include/CalculateIntersections.h

```diff
@@ -9,7 +9,7 @@
 
 namespace Math {
 
-    // Important case - it is when 2 lines not parallel and sections have intersection. If case is important, we get coords of intersection point, else we get (0,0) point
+// Important case - it is when 2 lines not parallel and sections have intersection. If case is important, we get coords of intersection point, else we get (0,0) point
```

Comment:
When 2 lines are not parallel and sections have an intersection, get coords of intersection point; otherwise get a (0,0) point